jobs/aqm/stats/JEVS_AQM_STATS

```diff
@@ -1 +1,84 @@
-example
+#!/bin/ksh
+
+########################################
+# Preliminary data setup step
+########################################
+set -xa
+
+# ###################################
+# SET SHELL PROCESSING VARIABLES
+# ###################################
+export PS4='$SECONDS + '
+date
+
+###########################################################
+# obtain unique LSF id (jobid) and make temp directories
+###########################################################
+export pid=$$
+#export DATA_IN=${DATA_IN:-$DATAROOT}
+#export DATA=$DATA_IN/${jobid}
+
+export DATA=/lfs/h2/emc/ptmp/$USER/EVS/${cyc}_${MODELNAME}
+
+rm -rf $DATA
+mkdir -p $DATA
+cd $DATA
+
+####################################
+# Determine Job Output Name on System
+####################################
+export pgmout="OUTPUT.${pid}"
+
+export cycle=t${cyc}z
+##################################################
+# SAVEGES  - Copy Files From TMPDIR to $GESdir
+# SENDECF  - Flag Events on ecFLOW
+# SENDCOM  - Copy Files From TMPDIR to $COMOUT
+# SENDDBN  - Issue DBNet Client Calls
+##################################################
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-YES}
+export SENDECF=${SENDECF:-YES}
+
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
+export EXECevs=${EXECevs:-$HOMEevs/exec}
+export FIXvevs=${FIXevs:-$HOMEevs/fix}
+export PARMevs=${PARMevs:-$HOMEevs/parm}
+export USHevs=${USHevs:-$HOMEevs/ush}
+
+# Run setpdy and initialize PDY variables
+setpdy.sh
+. ./PDY
+
+export VDATE=$PDYm3
+
+# Define COMIN/COMOUT variables:
+
+echo $COMPATH
```

Comment:
What is this? I don't see it defined in the J-job or upstream in the ecf script.